apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx

```diff
@@ -43,23 +43,23 @@ export const Member = () => {
                 </div>
 
                 <div className="flex w-full items-end gap-2">
-                    <div className="w-[75%]">
+                    <div className="flex-1">
                         <TextInput
                             placeholder=""
                             value={addEllipsis(inviteLink, 60)}
                             disabled={true}
                             rightIcon={
                                 <button disabled={loading} onClick={handleInviteLinkRefresh}>
```

Comment:
@afolasope What is the point of the custom `Button` component we have if we still use the native component.